Framework/Core/include/Framework/TableBuilder.h

```diff
@@ -15,6 +15,7 @@
 #include "Framework/StructToTuple.h"
 #include "Framework/FunctionalHelpers.h"
 #include "Framework/VariantHelpers.h"
+#include "Framework/Logger.h"
```

Comment:
Please do not expose Logger.h in public headers.

Reply:
Forgot to remove after debugging, removed now.